src/main/java/ro/msg/learning/shop/dto/ViewOrderDto.java

```diff
@@ -0,0 +1,19 @@
+package ro.msg.learning.shop.dto;
+
+import lombok.Builder;
+import lombok.Data;
+import ro.msg.learning.shop.domain.Customer;
+
+import java.time.LocalDateTime;
+
+@Data
+@Builder
+public class ViewOrderDto {
+    private String id;
+    private Customer customer;
```

Comment:
`Customer` is an entity. Should not be in the DTO. Instead, you can return only the `customerName`, for example.